ormconfig-metadata.js

```diff
@@ -0,0 +1,28 @@
+const { SnakeNamingStrategy } = require('typeorm-naming-strategies');
```

Comment:
Out of curiosity, why do we need this metadata config file? could not live in the current form config file?

Reply:
The "main" TypeORM migrations are for the tables we automatically put into every client database, while this is now metadata for the engine itself to manage the databases in its own private database, so we don't want those tables to show up in every user's databases.

Reply:
oh so I think this answers part of my question [here](https://github.com/iasql/iasql-engine/pull/542#pullrequestreview-915462650) but how do we handle persistence? where are we creating and keeping our own private db with this info? I think that part is what I do not follow

Reply:
> oh so I think this answers part of my question [here](https://github.com/iasql/iasql-engine/pull/542#pullrequestreview-915462650) but how do we handle persistence? where are we creating and keeping our own private db with this info? I think that part is what I do not follow

That's why I asked about the accidental usage of `__example__` as the database name in another of my comments. :)

Our RDS database will have one database only accessible to the root account that we store this metadata in. (Similarly for anyone running it locally, it will be that same database name in their local postgres docker container.)

Basically we will have one singleton database that is for the engine itself and not for the users of the engine.